### PR TITLE
Update UG and DG

### PR DESCRIPTION
**Pull request by Ellevy**
UG changes: Changed intro and names
DG changes: Target user and value proposition + user stories and use cases

### CODE REVIEW

**Review by pyuxiang — APPROVED** [review]
LGTM 👍🏻

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -236,221 +236,182 @@ _{Explain here how the data archiving feature will be implemented}_
...
+| `*` | Returning user | Mark celebrations as completed | Know which birthdays are done |
+| `**` | Returning user | Delete completed/irrelevant contacts | Reduce the clutter on PartyPlanet |
+| `***` | Retiring welfare IC | Pass down the data to successor | (they) would not need to re-gather and re-enter details |
+| `*` | Expert user | Create shortcuts | Run multiple repeated commands at once to save time |
+| `**` | Clumsy user | Edit misspelled commands | Fix typo mistakes in the app |
+| `**` | Clumsy user | Undo/edit misspelled names/numbers/notes | Fix mistakes |
+| `*` | Reflective user | Archive past birthday celebrations | Revisit previous birthday celebration and their details for reuse |
+| `*` | CCA welfare IC with many sub-groups | Color tags | Differentiate contacts easily |
+| `**` | Welfare IC who plans ahead of time | Search date | See all birthdays on that day |
+| `* *` | Night owl | Enable dark mode | Use the app safely in dark environments |
```
Comment (nickyfoo) [inline docs/DeveloperGuide.md:280]:
```suggestion
| `**` | Night owl | Enable dark mode | Use the app safely in dark environments |
```

**Review by nickyfoo — CHANGES_REQUESTED** [review]
apart from small typo LGTM

**Review by pyuxiang — APPROVED** [review]
Another LGTM because I dismissed my own review.

**Review by nickyfoo — APPROVED** [review]
LGTM

**Pull request merged**